charts/clusterpedia/values.yaml

```diff
@@ -41,6 +41,14 @@ commonAnnotations: {}
 ## if PV resources are not required, set it to None!
 ## if defined, will create the pv on the node.
 persistenceMatchNode: ""
+## @param telemetry define flag whether to enable otel trace
+##
+telemetry:
```

Comment:
This configuration should only work with apiserver, right?

Reply:
currentlly it only works with apiserver

Reply:
Done. change it to apiserver.telemetry